src/PKSim.Core/Model/IndividualSimulation.cs

```diff
@@ -40,25 +37,7 @@ public virtual DataRepository DataRepository
 
       public int IndividualId => Results.AllIndividualIds().FirstOrDefault();
 
-      public override TBuildingBlock BuildingBlock<TBuildingBlock>()
-      {
-         return AllBuildingBlocks<TBuildingBlock>().SingleOrDefault();
-      }
-
-      public override void UpdatePropertiesFrom(IUpdatable sourceObject, ICloneManager cloneManager)
```

Comment:
I am not sure why those guys were defined an empty. Probably after some refactoring. Removed